types/xrm/xrm-tests.ts

```diff
@@ -609,10 +609,19 @@ Xrm.Navigation.navigateTo({
     },
 );
 
+const optionSetControl = Xrm.Page.getControl<Xrm.Controls.OptionSetControl>("singlechoice");
```

Comment:
Can you please move this line closer to its actual usage instead of splitting it. Thanks.

Reply:
Done. 

If you don't have much time on your hands, please feel free to ignore the rest of this comment :D


Just out of curiosity, since I'm not very familiar with this kind of tests:

Should I create my own objects in the tests file, or reuse existing ones?
`multiSelectOptionSetControl` had a very generic name, which made it seem like a good idea to reuse it.

Would it theoretically be better to create a function like this in xrm-tests, instead of appending to the script scope:

```typescript
function testGetOptions(formContext: Xrm.FormContext) {
    const optionSetControl = formContext.getControl<Xrm.Controls.OptionSetControl>("singlechoice");
    // ... Rest of the test goes here
}
```

This introduces more granular scoping and ties creation of objects to the actual test case. Additionally, using Xrm.Page wouldn't be required anymore.